build.gradle

```diff
@@ -1,29 +1,26 @@
 plugins {
-    id "java-library"
-    id "scala"
-    id "maven-publish"
-    id 'fabric-loom' version '0.10.30'
-    id "com.matthewprenger.cursegradle" version "1.4.0"
+	id 'java-library'
+	id 'scala'
+	id 'maven-publish'
+	id 'fabric-loom' version '1.5-SNAPSHOT'
 }
 
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
-
 archivesBaseName = project.archives_base_name
 group = project.maven_group
 
-version = mod_version
-def ENV = System.getenv()
-version = version + "+scala." + project.scala_version + (ENV.GITHUB_ACTIONS ? "" : ".local")
+version = mod_version + "+scala." + project.scala_version + (System.getenv('GITHUB_ACTIONS') ? "" : ".local")
 
 dependencies {
-    minecraft "com.mojang:minecraft:${project.minecraft_version}"
-    mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	minecraft "com.mojang:minecraft:${project.minecraft_version}"
+	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
 	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
-	include(api("org.scala-lang:scala3-library_3:${project.scala_version}"))
-	// Depended on by scala3
-	include(api("org.scala-lang:scala-library:2.13.6"))
+	// Scala Dependencies
+	modApi(include("org.scala-lang:scala3-library_3:${project.scala_version}"))
+	// Dependent by Scala3
+	modApi(include("org.scala-lang:scala-library:2.13.12"))
```

Comment:
This scala-library version should be moved alongside the `scala_version I also see there may be a newer version of this: [3.4.0](https://mvnrepository.com/artifact/org.scala-lang/scala3-library_3/3.4.0) is the 2.x version correct?

Reply:
`org.scala-lang:scala-library:2.13.12` was required by Scala3's standard library, only this version.

Reply:
Scala 3.4.0 is not supported well by the Scala ecosystem, so I think Scala 3.3.3 is the best (since it's the LTS version)

Reply:
Thats great, I was just a little confused as to why Scala 3.3.3 was depending on a 2.x version of the library. Thanks for confirming.

Reply:
oh, this is because scala3 has to be very compatible with scala2, so scala3 will depend on scala2 libraries